website/docs/guides/parameter-validation-script.md

```diff
@@ -1,24 +1,24 @@
-# Parameters when triggering an ORT run
+# Parameter Validation Script
 
 This document describes the mechanism how the parameters of an ORT run can be specified when invoking the corresponding API endpoint and how this can be customized.
 
 ## Problem
 
 In the data model of ORT Server, so-called _job configurations_ are used to specify the exact behavior of workers, like the Analyzer, the Scanner, etc. While some parts of such configurations are rather straight-forward, e.g. flags like `skipExcluded`, others are not and partly require a deep technical knowledge about the runtime environment and how it is configured. To give a few examples:
 
-* When specifying the list of _advisors_ to run, their configuration is expected. This includes information like URLs of external systems and access credentials.
-* Some workers rely on plugins that implement the actual functionality. Those plugins expect their configuration in maps containing key-value pairs; and the whole plugin configuration is a map that uses the plugin names as keys and the plugin options as values. So, the name of plugins and their supported options must be known.
-* The rules to be processed by the Evaluator must be loaded from somewhere. Where are they located and how can they be accessed?
-* The situation for template files to be consumed by the Reporter is similar; they also need to be retrieved from a specific location.
+- When specifying the list of _advisors_ to run, their configuration is expected. This includes information like URLs of external systems and access credentials.
```

Comment:
Strictly speaking, changing the bullet list markers is unrelated to moving files around.

Reply:
The reformatting is required because the website uses prettier for the formatting like the ui does. I have added this to the commit message now.